src/sentry/integrations/discord/actions/issue_alert/notification.py

```diff
@@ -49,7 +48,7 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
             client = DiscordClient()
             try:
                 client.send_message(channel_id, message, notification_uuid=notification_uuid)
-            except ApiError as e:
+            except Exception as e:
```

Comment:
This extra info was not being logged since the error was not ApiError

Reply:
Same with the send_message in metric_alert.py

Reply:
The base client class has some control over the error type, which is why we might not have been catching all errors: https://github.com/getsentry/sentry/blob/bb1a2ef98d0b9d4b5fb77c38daa8511bf1b20596/src/sentry/shared_integrations/client/base.py#L252-L300